src/Playroom/Snippets/Snippets.tsx

```diff
@@ -159,13 +160,12 @@ export default ({ snippets, onHighlight, onClose }: Props) => {
               closeHandler(null);
             }
           }}
-          data-testid="filterSnippets"
         />
       </div>
       <ul
         className={styles.snippetsContainer}
-        data-testid="snippet-list"
         ref={listEl}
+        aria-label="Filtered snippets list"
```

Comment:
Not sure if there are conventions for labeling lists, but given that the element is a `ul`, adding `list` to the label itself seems a bit redundant.

```suggestion
        aria-label="Filtered snippets"
```